### ISSUE TITLE
How about the loss converges during training?

### ISSUE THREAD

**Issue by Vickeyhw** [facebookresearch/moco-v3#23]
During training, I find that the training loss is not monotonically decreasing, is it right? Does the loss number indicate the training situation? If not, when the pretraining finished, how much is the samples matching accuracy should be?

**Comment by endernewton** [comment]
Could you share a screenshot to give more context? Due to instability, it will generally have dips (or spikes) in loss.

**Comment by Bourne-M** [comment]
> Could you share a screenshot to give more context? Due to instability, it will generally have dips (or spikes) in loss.

![image](https://user-images.githubusercontent.com/13611481/151477634-3047c3c4-e1c0-486e-b414-fad55a4c70d5.png)
is this normal?

**Comment by endernewton** [comment]
Looks pretty normal to me. The loss is generally going down. closing as this is more about batch-wise variations.

**Comment by Vickeyhw** [comment]
![1645366416(1)](https://user-images.githubusercontent.com/20540333/154846931-bbfe8a8b-d781-4847-8c70-6ad090cd5816.png)
I mean that after several epochs, the loss increases, is it normal?
@endernewton

**Comment by ayumiymk** [comment]
Hi, I have a similar issue while training on my own datasets. Do you find any solutions? @Vickeyhw
